ff/serving/metadata-rest/metadata-rest-server.go

```diff
@@ -0,0 +1,611 @@
+package main
```

Comment:
package names shouldnt have a -, go files should use underscore:
I would make this a subpackage of metadata (so a subdirectory of metadata called dashboard)
ff/serving/metadata/dashboard

I'd rename the file to dashboard_metadata.go